src/redux/tokens/actions.ts

```diff
@@ -0,0 +1,23 @@
+import { IAccountState } from '../wallets/state';
+import { ITokenConfigState } from './state';
+import { Dispatch } from 'react';
+import { IReduxState } from '../state';
+import { addTokenToAccount } from '../wallets/actions';
+import { convertTokenConfig } from './static-selectors';
+import { getChainId } from '../preferences/selectors';
+
+export const ADD_TOKEN = 'ADD_TOKEN';
+export const REMOVE_TOKEN = 'REMOVE_TOKEN';
+
+export const addToken = (account: IAccountState, token: ITokenConfigState) => (
+    dispatch: Dispatch<any>,
+    getState: () => IReduxState
+) => {
+    const blockchain = account.blockchain;
+    const chainId = getChainId(getState(), blockchain).toString();
+    dispatch({
+        type: ADD_TOKEN,
+        data: { token, chainId, blockchain }
+    });
+    addTokenToAccount(account, convertTokenConfig(token, account))(dispatch, getState);
```

Comment:
`convertTokenConfig` is returning a default order of `0`, when adding a token we should add it at the end of the list, so you can use `order: 999`

Reply:
convertTokenConfig(generateAccountTokenState) has a optional account, if the account is given then the order returned is the number of tokens + 1 so each time a new token is added it will be the last